src/services/authService.ts

```diff
@@ -7,15 +7,6 @@ export async function login(email: string, password: string): Promise<string> {
   console.log(email)
```

Comment:
remove console.logs

add underline when you are not using parameters in the function login(_email: string, _password: string) to avoid errors.

Reply:
Done, didn't know that was a thing. Quite a nifty trick haha.